ctapipe/containers.py

```diff
@@ -570,6 +574,82 @@ class TelescopeImpactParameterContainer(Container):
     distance_uncert = Field(nan * u.m, "uncertainty in impact_parameter", unit=u.m)
 
 
+class PixelMonitoringContainer(Container):
+    """
+    True information about nsb rate, quantum efficiency, high voltage, current and amplification settings for each pixel and telescope.
+    """
+
+    nsb_pe_rate = Field(np.int32(-1), "NSB pixel p.e. rate in units of p.e./ns")
+
+    qe_rel = Field(None, "Quantum efficiency w.r.t nominal")
+
+    high_voltage = Field(None, "High voltage w.r.t. nominal")
+    current = Field(None, "Current at pixel")
+    fadc_amp = Field(None, "FADC amplitude per mean per p.e.")
+    gain = Field(None, "Assumed (PMT/common) gain w.r.t. nominal")
+
+
+class LaserCalibrationContainer(Container):
+    """
+    Laser calibartion information computed
+    """
+
+    calib = Field(None, "ADC to laser/LED p.e.")
+    max_integ_frac = Field(
+        None,
+        "Maximum fraction of the signal which can be in the fixed integration window",
+    )
+    max_timing_frac = Field(
+        None,
+        "Maximum fraction of the signal which can be in the pixel timing integration",
+    )
+    tm_calib = Field(nan * u.ns, "Transit time calibration", unit=u.ns)
+    flat_fielding = Field(None, "Flat-field correction")
+
+
+class CameraMonitoringContainer(Container):
+    """
+    Camera monitoring
+    """
+
+    monitor_id = Field(-1, "Monitoring id incremented with each update")
+    monitor_time = Field(None, "Time when last monitoring data was sent")
+    status_time = Field(None, "Status time")
+    trigger_time = Field(None, "Time when last trigger monitor data was read")
+    trigger_rate = Field(None, "Camera average local trigger rate in Hz")
```

Comment:
I'd suggest adding units to all of these that are applicable rather than mentioning the unit in the help string, e.g.  
```py3
trigger_rate = Field(None, "Camera average local trigger rate", unit=u.s**-1)
data_rate = Field(-1, "Average rate of packed data", unit=u.Mbyte/u.s)
drawer_temp = Field(None, "Drawer temperature", unit=u.deg_C)
```